frontend/src/metabase/query_builder/components/dataref/TablePane.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { connect } from "react-redux";
+import { t } from "ttag";
+import _ from "underscore";
+
+import { State } from "metabase-types/store";
+import Tables from "metabase/entities/tables";
+import {
+  Description,
+  EmptyDescription,
+} from "metabase/components/MetadataInfo/MetadataInfo.styled";
+import ConnectedTableList from "metabase/query_builder/components/dataref/ConnectedTableList";
+import type Table from "metabase-lib/lib/metadata/Table";
+import TableInfoLoader from "./TableInfoLoader";
+import FieldList from "./FieldList";
+
+const mapStateToProps = (state: State, props: TablePaneProps) => ({
+  table: Tables.selectors.getObject(state, {
+    entityId: props.table.id,
+  }),
+});
+
+interface TablePaneProps {
+  show: (type: string, item: unknown) => void;
+  table: Table;
+}
+
+const TablePane = ({ table, show }: TablePaneProps) => {
+  return table ? (
+    <TableInfoLoader table={table}>
+      <>
```

Comment:
We can probably drop this Fragment, no ?
The `<>` and `<\>` on line 53?

Reply:
Yes, fixed here https://github.com/metabase/metabase/pull/25751/commits/c1a2c6a5cb2e3945c37c39a00d8b9813d99c8d97